projects/plugins/jetpack/_inc/client/components/popover/index.jsx

```diff
@@ -193,15 +203,23 @@ class Popover extends Component {
 
 	onClickout( event ) {
 		let shouldClose =
-			this.domContext && this.domContext.contains && ! this.domContext.contains( event.target );
+			this.domContextRef.current &&
+			this.domContextRef.current.contains &&
```

Comment:
This can likely be an optional chain:

```suggestion
			this.domContextRef.current?.contains &&
```

Reply:
Nice, added in [e44145f](https://github.com/Automattic/jetpack/pull/38339/commits/e44145f41a553458d5647fc60cd0670e300c00a1)